### PR TITLE
DDF-2824 Update IdP docs

### PR DESCRIPTION
**Pull request by brendan-hofmann**
#### What does this PR do?
Reflect the default installation of the IdP and fix some typos.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@ricklarsen @codymacdonald 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Docs](https://github.com/orgs/codice/teams/docs)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@clockard
@coyotesqrl
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)
#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
build now

File: distribution/docs/src/main/resources/_contents/_securing/securing-idp-sp-contents.adoc [hunk 1]
```diff
@@ -9,8 +9,7 @@ IdP and SP are used for SSO authentication purposes.
 
 ===== Installing the IdP
 
-The IdP bundles are not installed by default.
-They can be started by installing the `security-idp` feature.
+The IdP bundles are installed by default with ${ddf-security}.
```
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/_securing/securing-idp-sp-contents.adoc]: can you edit to say ` with the ${ddf-security} application`?

**Comment by clockard** [comment]
build now

File: distribution/docs/src/main/resources/_contents/_securing/securing-idp-sp-contents.adoc (hunk 1 above)
Comment (brendan-hofmann) [inline distribution/docs/src/main/resources/_contents/_securing/securing-idp-sp-contents.adoc]: No problem, fixed.

**Pull request merged**